### PR TITLE
Update manifests to reduce required permissions

### PR DESCRIPTION
**Pull request by filariow**
depends on #90

Signed-off-by: Francesco Ilario <filario@redhat.com>

### CODE REVIEW

File: config/rbac/role_binding.yaml [hunk 1]
```diff
@@ -9,11 +9,11 @@ metadata:
     app.kubernetes.io/part-of: primaza
     app.kubernetes.io/managed-by: kustomize
   name: manager-rolebinding
+  namespace: system
```
Comment (dperaza4dustbit) [inline config/rbac/role_binding.yaml:12]: Where does system come from is this supposed to be a variable with the namespace for this primaza tenant?

Comment (filariow) [reply config/rbac/role_binding.yaml:12]: That's just a placeholder that can be used in kustomize manifests to identify objects. Kustomize's default seemed to be `system` so we are relying on this value. We can choose a different value if we prefer.

Default usage of these manifests would rely on overwriting this value.

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

**Pull request merged**